src/nls/nl/strings.js

```diff
@@ -265,45 +383,50 @@ define({
     "CMD_TOGGLE_WORD_WRAP"                : "Word wrap",
     "CMD_LIVE_HIGHLIGHT"                  : "Live voorbeeld markeren",
     "CMD_VIEW_TOGGLE_INSPECTION"          : "Lint bestanden bij opslaan",
-    "CMD_SORT_WORKINGSET_BY_ADDED"        : "Sorteren op toegevoegd",
-    "CMD_SORT_WORKINGSET_BY_NAME"         : "Sorteren op naam",
-    "CMD_SORT_WORKINGSET_BY_TYPE"         : "Sorteren op type",
-    "CMD_SORT_WORKINGSET_AUTO"            : "Automatisch sorteren",
-
+    "CMD_WORKINGSET_SORT_BY_ADDED"        : "Sorteren op toegevoegd",
+    "CMD_WORKINGSET_SORT_BY_NAME"         : "Sorteren op naam",
+    "CMD_WORKINGSET_SORT_BY_TYPE"         : "Sorteren op type",
+    "CMD_WORKING_SORT_TOGGLE_AUTO"        : "Automatisch sorteren",
+    "CMD_THEMES"                          : "Thema\u0027s\u2026",
+    
     // Navigate menu Commands
     "NAVIGATE_MENU"                       : "Navigeer",
     "CMD_QUICK_OPEN"                      : "Snel openen",
-    "CMD_GOTO_LINE"                       : "Ga naar regel",
+    "CMD_GOTO_LINE"                       : "Ga naar regel\u2026",
     "CMD_GOTO_DEFINITION"                 : "Definitie snel zoeken",
     "CMD_GOTO_FIRST_PROBLEM"              : "Ga naar de eerstvolgende fout/waarschuwing",
     "CMD_TOGGLE_QUICK_EDIT"               : "Snel wijzigen",
     "CMD_TOGGLE_QUICK_DOCS"               : "Snel naar documentatie",
     "CMD_QUICK_EDIT_PREV_MATCH"           : "Vorige overeenkomst",
     "CMD_QUICK_EDIT_NEXT_MATCH"           : "Volgende overeenkomst",
+    "CMD_CSS_QUICK_EDIT_NEW_RULE"         : "Nieuwe regels",
     "CMD_NEXT_DOC"                        : "Volgend document",
     "CMD_PREV_DOC"                        : "Vorig document",
     "CMD_SHOW_IN_TREE"                    : "Toon in bestandsboom",
+    "CMD_SHOW_IN_EXPLORER"                : "Toon in Explorer",
```

Comment:
'Explorer' could be renamed to 'Verkenner'. Dutch translation of the Windows file manager.